stdlib/2/stringold.pyi

```diff
@@ -0,0 +1,46 @@
+# Source: https://hg.python.org/cpython/file/2.7/Lib/stringold.py
+from typing import AnyStr, Iterable, List, Optional, Type
+
+whitespace = ...  # type: str
+lowercase = ...  # type: str
+uppercase = ...  # type: str
+letters = ...  # type: str
+digits = ...  # type: str
+hexdigits = ...  # type: str
+octdigits = ...  # type: str
+_idmap = ...  # type: str
+_idmapL = ...  # type: Optional[List[str]]
+index_error = ...  # type: Type[ValueError]
+atoi_error = ...  # type: Type[ValueError]
+atof_error = ...  # type: Type[ValueError]
+atol_error = ...  # type: Type[ValueError]
+
+
+def lower(s: AnyStr) -> AnyStr: ...
+def upper(s: AnyStr) -> AnyStr: ...
+def swapcase(s: AnyStr) -> AnyStr: ...
+def strip(s: AnyStr) -> AnyStr: ...
+def lstrip(s: AnyStr) -> AnyStr: ...
+def rstrip(s: AnyStr) -> AnyStr: ...
+def split(s: AnyStr, sep: AnyStr = ..., maxsplit: int = ...) -> List[AnyStr]: ...
```

Comment:
This (and most functions below) will probably also allow mixed str and unicode arguments.